src/main/java/org/isf/sms/gui/SmsEdit.java

```diff
@@ -283,55 +286,56 @@ private JPanel getJButtonPanel() {
 		}
 		return jButtonPanel;
 	}
-	
+
 	private JButton getJOkButton() {
 		if (jOkButton == null) {
 			jOkButton = new JButton(MessageBundle.getMessage("angal.common.ok.btn"));
 			jOkButton.setMnemonic(MessageBundle.getMnemonic("angal.common.ok.btn.key"));
 			jOkButton.addActionListener(new ActionListener() {
-				
+
 				public void actionPerformed(ActionEvent e) {
 					String number = jNumberTextField.getText().replaceAll(" ", "").trim(); //$NON-NLS-1$ //$NON-NLS-2$
 					String text = jTextArea.getText();
 					Date schedDate = jSchedDateChooser.getDate();
-					
+
 					Sms smsToSend = new Sms();
 					smsToSend.setSmsNumber(number);
 					smsToSend.setSmsDateSched(schedDate);
 					smsToSend.setSmsUser(UserBrowsingManager.getCurrentUser());
+					System.err.println("SMS_USER: " + UserBrowsingManager.getCurrentUser());
 					smsToSend.setSmsText(text);
 					smsToSend.setModule("smsmanager");
-					
+
 					try {
-						
+
 						smsManager.saveOrUpdate(smsToSend, false);
-						
+
 					} catch (OHServiceException e1) {
-						
+
 						if (e1.getMessages().get(0).getTitle().equals("testMaxLenghtError")) {
-							
+
 							int textLength = text.length();
 							int textParts = (textLength + maxLength - 1) / maxLength;
 							StringBuilder message = new StringBuilder();
-							message.append(e1.getMessages().get(0).getMessage())
-								.append("\n")
-								.append(MessageBundle.getMessage("angal.sms.doyouwanttosplitinmoremessages"))
-								.append(" (").append(textParts).append(")?");
-							
+							message.append(e1.getMessages().get(0).getMessage()).append("\n")
+											.append(MessageBundle.getMessage("angal.sms.doyouwanttosplitinmoremessages")).append(" (").append(textParts)
+											.append(")?");
+
```

Comment:
unnecessary new lines

Reply:
I restored the current informatici version